lib/rock_rms/resources/person.rb

```diff
@@ -53,6 +53,22 @@ def create_person(first_name:, last_name:, email:, connection_status_value_id: n
         post(people_path, options)
       end
 
+      def create_business(last_name:, email:, connection_status_value_id: nil, record_status_value_id: nil, record_type_value_id: 2)
```

Comment:
Probably should call the parameter `business_name` or just `name` and map it to `LastName` on 59.